tests/test_db.rs

```diff
@@ -513,6 +513,46 @@ fn test_open_as_secondary() {
     assert_eq!(get_byte_slice(&result), b"value2");
 }
 
+#[test]
+fn test_open_cf_descriptors_as_secondary() {
+    let primary_path = DBPath::new("_rust_rocksdb_test_open_cf_descriptors_as_secondary_primary");
+    let mut primary_opts = Options::default();
+    primary_opts.create_if_missing(true);
+    primary_opts.create_missing_column_families(true);
+    let cfs = vec!["cf1"];
+    let primary_db = DB::open_cf(&primary_opts, &primary_path, cfs.clone()).unwrap();
+    let primary_cf1 = primary_db.cf_handle("cf1").unwrap();
+    primary_db.put_cf(&primary_cf1, b"k1", b"v1").unwrap();
+
+    let secondary_path =
+        DBPath::new("_rust_rocksdb_test_open_cf_descriptors_as_secondary_secondary");
+    let mut secondary_opts = Options::default();
+    secondary_opts.set_max_open_files(-1);
+    let cfs = cfs.into_iter().map(|name| {
+        ColumnFamilyDescriptor::new(<str as AsRef<str>>::as_ref(name), Options::default())
```

Comment:
```suggestion
        ColumnFamilyDescriptor::new(name, Options::default())
```

Reply:
Glad these caught your eye - I initially (incorrectly) used `name.as_ref()` and then lazily followed the compiler's suggestion when it told me I needed type annotations.